source/ios/tools/AdaptiveCards.podspec

```diff
@@ -1,20 +1,25 @@
-  
-Pod::Spec.new do |spec|
+od::Spec.new do |spec|
   spec.name             = 'AdaptiveCards'
 
-  spec.version          = '2.8.2'
-  
+  spec.version          = '2.8.3'
```

Comment:
I see that you've already tagged 2.8.3 - https://github.com/microsoft/AdaptiveCards/releases/tag/2.8.3
but to a different commit. Was that intentional?

Reply:
yes, the hash is for testing.